addons/mrp/views/mrp_workorder_views.xml

```diff
@@ -168,9 +169,9 @@
                             <field name="move_id" invisible="1"/>
```

Comment:
if you have some workorder lines and you're in strict consumption, you can still add a line, can you check an error is properly raised?